ruff.toml

```diff
@@ -40,8 +40,8 @@ select = [
 "adodbapi/*" = ["C4", "YTT301", "UP031", "UP032", "ISC002"]
 
 [lint.isort]
-combine-as-imports = true
 # Because pywin32 has a mix of relative and absolute imports, with undetectable first-party c-extensions
+combine-as-imports = true
```

Comment:
```suggestion
combine-as-imports = true
# Because pywin32 has a mix of relative and absolute imports, with undetectable first-party c-extensions
```